gk/main.c

```diff
@@ -1408,6 +1413,20 @@ update_ip_hop_count(struct gatekeeper_if *iface, struct ipacket *packet,
 	return 0;
 }
 
+static void
+send_request_to_grantor(struct ipacket *packet, struct gk_fib *fib,
+		struct gk_measurement_metrics *stats,
+		struct gk_instance *instance, struct gk_config *gk_conf) {
```

Comment:
Drop the parameter `stats` and use `&instance->traffic_stats` in the single place where it's needed.